README.md

```diff
@@ -45,9 +50,9 @@ If you do not want errors to be caught while the page is loading, [use this snip
 
 ### Synchronous methods
 
-Note that using these methods will not catch errors thrown while the page is loading. The script needs to be referenced before your other site/app scripts, and will block the page load while it is being downloaded/parsed/executed.
+Note that using these methods will not catch errors thrown while the page is loading. The script needs to be referenced before your other site/app scripts and will block the page load while it is being downloaded/parsed/executed.
 
-This will also disrupt RUM timings, making them erroneous. For RUM, it is especially importing that the async snippet method above is used, instead of one of the following.
+This will also disrupt RUM timings, making them erroneous. For RUM, it is especially important that the async snippet method above is used, instead of one of the following.
 
```

Comment:
Do we need to add a section here - 
```
#### With NPM
Run `npm install raygun4js --save
```

And then we can also remove the section on line 78-80 to remove a double up

Reply:
I think it looks okay as is because React Native still needs clarity on where to install from and it seems like NPM over Bower is the case here :)